spacelift/resource_security_email.go

```diff
@@ -0,0 +1,87 @@
+package spacelift
+
+import (
+	"context"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/spacelift-io/terraform-provider-spacelift/spacelift/internal"
+)
+
+func resourceSecurityEmail() *schema.Resource {
+	return &schema.Resource{
+		Description: "" +
+			"`spacelift_security_email` represents a security email address that " +
```

Comment:
```suggestion
			"`spacelift_security_email` represents an email address that " +
```

Just a nit - I don't think we need `security` in here twice.